lib/syslogger.rb

```diff
@@ -126,7 +126,7 @@ def max_octets=(max_octets)
   def level=(level)
     level = Logger.const_get(level.to_s.upcase) if level.is_a?(Symbol)
 
-    unless level.is_a?(Fixnum)
+    unless level.is_a?(Integer)
```

Comment:
@pjg How so?  `Integer` class is even in [1.8.7](http://ruby-doc.org/core-1.8.7/Integer.html).